app/src/test/kotlin/com/x8bit/bitwarden/data/vault/datasource/sdk/model/CipherListViewUtil.kt

```diff
@@ -110,12 +110,14 @@ fun createMockFido2CredentialListView(
     userHandle: String = "mockUserHandle-$number",
     userName: String = "mockUserName-$number",
     userDisplayName: String = "mockUserDisplayName-$number",
+    counter: String = "0",
```

Comment:
Can we change this to parameter to `hasCounter` and assign `counter = if (hasCounter) "$number" else "0"`?